### ISSUE TITLE
Fatal Exception: java.lang.NoClassDefFoundError: Failed resolution of: Ljava/time/Instant; - Android < 8.0

### ISSUE THREAD

**Issue by Andr3Carvalh0** [SourcePointUSA/android-cmp-app#575]
**Describe the bug**
When using the library on android 8.0 it crashes when trying to parse dates

** About your environment **
 - SDK version: 7.0.0
 - Device: Android
 - OS: < 8.0

**To Reproduce**
1. Use a configuration/json that has a date object in it.
2. Try to load it. As soon as the library starts to parse the JSON it will crash

**Additional context**
```Fatal Exception: java.lang.NoClassDefFoundError
Failed resolution of: Ljava/time/Instant;
com.sourcepoint.cmplibrary.data.network.converter.DateSerializer.deserialize (DateSerializer.java)
com.sourcepoint.cmplibrary.data.network.converter.DateSerializer.deserialize (DateSerializer.java)
kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic (PolymorphicKt.java:221)
kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue (StreamingJsonDecoder.java)
kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue (AbstractDecoder.java)
com.sourcepoint.cmplibrary.data.network.model.optimized.MetaDataResp$Gdpr$$serializer.deserialize (MetaDataResp.java)
com.sourcepoint.cmplibrary.data.network.model.optimized.MetaDataResp$Gdpr$$serializer.deserialize (MetaDataResp.java)
kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic (PolymorphicKt.java:221)
kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue (StreamingJsonDecoder.java)
kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue (AbstractDecoder.java)
com.sourcepoint.cmplibrary.data.network.model.optimized.MetaDataResp$$serializer.deserialize (MetaDataResp.java)
com.sourcepoint.cmplibrary.data.network.model.optimized.MetaDataResp$$serializer.deserialize (MetaDataResp.java)
kotlinx.serialization.json.internal.PolymorphicKt.decodeSerializableValuePolymorphic (PolymorphicKt.java:221)
kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue (StreamingJsonDecoder.java)
kotlinx.serialization.json.Json.b (Json.java:21)
com.sourcepoint.cmplibrary.campaign.CampaignManagerImpl$getMessageOptimizedReq$6$1$$InternalSyntheticOutline$328$15f73016d045466bf791c07683502dab54f4228bd47a56d2706b111404d5fee5$0.m (CampaignManagerImpl.java:2)
com.sourcepoint.cmplibrary.data.network.converter.JsonConverterImpl$toMetaDataRespResp$1.invoke (JsonConverterImpl.java:13)
kotlinx.serialization.json.Json.getSerializersModule (Json.java)
com.sourcepoint.cmplibrary.util.FunctionalUtilsKt.check (FunctionalUtilsKt.java)
com.sourcepoint.cmplibrary.data.network.converter.JsonConverterImpl.toMetaDataRespResp (JsonConverterImpl.java)
com.sourcepoint.cmplibrary.data.network.util.ResponseManagerImpl.parseMetaDataRes (ResponseManagerImpl.java:62)
com.sourcepoint.cmplibrary.data.network.NetworkClientImpl$getMetaData$1.invoke (NetworkClientImpl.java:70)
okhttp3.HttpUrl.toString (HttpUrl.java)
com.sourcepoint.cmplibrary.util.FunctionalUtilsKt.check (FunctionalUtilsKt.java)
com.sourcepoint.cmplibrary.data.network.NetworkClientImpl.getMetaData (NetworkClientImpl.java)
com.sourcepoint.cmplibrary.data.ServiceImpl.getMetaData (ServiceImpl.java)
com.sourcepoint.cmplibrary.data.ServiceImpl$getMessages$1.invoke (ServiceImpl.java)
com.sourcepoint.cmplibrary.data.ServiceImpl$getMessages$1.invoke (ServiceImpl.java)
com.sourcepoint.cmplibrary.core.ExecutorManagerImpl$executeOnWorkerThread$1.invoke$lambda-0 (ExecutorManagerImpl.java)
com.sourcepoint.cmplibrary.core.ExecutorManagerImpl$executeOnWorkerThread$1.$r8$lambda$YcU9LB-LQlcZOnRwBNhL6k0AYg0 (ExecutorManagerImpl.java)
com.sourcepoint.cmplibrary.core.ExecutorManagerImpl$executeOnWorkerThread$1$$InternalSyntheticLambda$1$270b46b0f6bf75c77303e1f25d99cf384b0205b066c32f1b52d0c4bc82f1eb7d$0.run (ExecutorManagerImpl.java)```

**Comment by carmelo-iriti** [comment]
Hi @Andr3Carvalh0,

thanks for the message, 

we just released the v7.0.2 which contains a fix to this issue, it'll be available in a couple of hours

**Comment by Andr3Carvalh0** [comment]
@carmelo-iriti Seems fixed, thanks!
